Documentation/docs-mobile/building-apps/build-properties.md

```diff
@@ -1079,6 +1079,15 @@ Specifies the name of the Resource
 file to generate. The default template sets this to
 `Resource.designer.cs`.
 
+## AndroidResourceDesignerClassModifier
+
+Specifies the class modifier for the intermediate `Resource` class which is
+generated. Valid values are `public`, `internal` and `private`.
```

Comment:
Should we remove mention of `private` in the markdown file? Since the code below does:
```xml
   FormatArguments="AndroidResourceDesignerClassModifier;$(AndroidResourceDesignerClassModifier);&apos;public&apos;, &apos;internal&apos;"
```